docs/source/interfaces/DataProcessorWidget.rst

```diff
@@ -50,7 +50,7 @@ Below is an example illustrating how to create a white list and add some columns
 
 .. code-block:: python
 
-    whitelist = MantidQt.MantidWidgets.DataProcessorWhiteList()
+    whitelist = MantidQt.MantidWidgets.WhiteList()
```

Comment:
Should this be `MantidQt.MantidWidgets.DataProcessor.WhiteList()` (here and in many other places below)? If I try to open the `ISIS SANS v2 experimental` interface I get the error:
```
AttributeError: type object 'MantidWidgets' has no attribute 'WhiteList'
  at line 28 in '<Interface>'
  caused by line 155 in '/media/mhb52947/DATAPART1/Users/mhb52947/work/checkout/linux/mantid_fork/scripts/Interface/ui/sans_isis/sans_data_processor_gui.py'
```